lib/resources/RecipientTransfers.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
+
+// This resource is only used when expanding a balance tranasaction
+// it can not be listed, retrieved or updated
```

Comment:
I don't think we need this file though. Resources should just be coming back as plain old objects:

```
$ npm run mocha debug -- test/stripe.spec.js --grep 'Will expose'

> stripe@4.16.1 mocha /Users/brandur/stripe/stripe-node
> mocha "debug" "test/stripe.spec.js" "--grep" "Will expose"

< Debugger listening on [::]:5858
connecting to 127.0.0.1:5858 ... ok
break in node_modules/mocha/bin/_mocha:7
  5  */
  6
> 7 var program = require('commander')
  8   , path = require('path')
  9   , fs = require('fs')
cont


<   Stripe Module
<     Callback support
<       Any given endpoint
break in test/stripe.spec.js:99
  97             },
  98           }, function(err, customer) {
> 99           debugger;
 100             cleanup.deleteCustomer(customer.id);
 101
debug> repl
Press Ctrl + C to leave debug repl
> customre
ReferenceError: customre is not defined
> customer
{ id: 'cus_ANyfLDmaBoZSJW',
  object: 'customer',
  account_balance: 0,
  created: 1490916606,
  currency: null,
  default_source: 'card_ANyfWchTFHrEWG',
  ...
```

Note how other subresource-only models are not defined (or as far as I can tell anyway).